framework/src/materials/ParsedMaterialHelper.C

```diff
@@ -51,6 +51,31 @@ ParsedMaterialHelper<is_ad>::ParsedMaterialHelper(const InputParameters & parame
 {
 }
 
+template <bool is_ad>
+void
+ParsedMaterialHelper<is_ad>::insertReservedNames(std::set<std::string> & reserved_names)
+{
+  for (const auto symbol : _extra_symbols)
+    switch (symbol)
```

Comment:
It seems like we never hit this method with a non-empty `_extra_symbols`. I would need to look into `ParsedMaterialHelper` more to know whether this is expected or not. I would think that `_extra_symbols` should always include the symbols below

Reply:
So always add "x", "y", "z", and "t" and ignore _extra_symbols? But I think like it is now, it is most "safe".

Reply:
If we always add "x", "y", "z", and "t", do we have to check in `ParsedMaterialHelper<is_ad>::ParsedMaterialHelper`  the param "extra_symbols" and error, if they are not present?

Reply:
I see now that `_extra_symbols` comes through the parameter `"extra_symbols"`. So I think the code as-is is good